internal/handlers/handlers.go

```diff
@@ -1 +1,61 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	data, err := os.ReadFile("..\\index.html")
```

Comment:
Нужно запускать из корня проекта, поэтому путь до файла будет такой
```suggestion
	data, err := os.ReadFile("index.html")
```